packages/usa-table/src/usa-table.stories.js

```diff
@@ -47,6 +52,11 @@ export const Sortable = SortableTemplate.bind({});
 Sortable.args = {
   ...SortableContent,
 };
+Sortable.argTypes = {
+  table_footer: {
+    table: { disable: true },
+  },
```

Comment:
This hides the table footer control from variants that do not have table footers in their templates